src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
-		printf("%s\n", strings[j]);
+		printf("callback: %s\n", strings[j]);
```

Comment:
[cppcoreguidelines-pro-bounds-pointer-arithmetic]: do not use pointer arithmetic